### ISSUE TITLE
[CS 1.6] Xash3D support! [ Feature Request ]

### ISSUE THREAD

**Issue by Karaulov** [ValveSoftware/halflife#2119]
Please add xash3d support ))) For play CS 1.6 in Android !

Or create port for android, please! :)))

**Comment by tyabus** [comment]
It already has been done, and ported to android. Look for cs16client.

**Comment by tyabus** [comment]
But cs 1.6 is boring crap, I don't recommend to play this degenerative game.

**Comment by Karaulov** [comment]
@tyabus but port not works with PC players (from Steam CS 1.6)

**Comment by tyabus** [comment]
Nope. You can compile cs16client v1.32, and modify engine to use mouse when sv_allow_mouse is 0
